modules/codegen/src/smithy4s/codegen/internals/Renderer.scala

```diff
@@ -32,11 +32,24 @@ import scala.jdk.CollectionConverters._
 import Line._
 import LineSyntax.LineInterpolator
 import ToLines.lineToLines
+import java.beans.Introspector.decapitalize
```

Comment:
TIL

Reply:
First time in years I used `java.beans` :joy: Knew something has to be there is stdlib, stackoverflow to the rescue :)

Reply:
we do have an `uncapitalise` method somewhere, but this is fine, I'll probably change it later on 😄

Reply:
Got it, updated